web/impact/impact/tests/test_graphql.py

```diff
@@ -56,6 +58,60 @@ def test_query_with_expert(self):
                                 'firstName': user.first_name,
                             },
                             'bio': user.expertprofile.bio,
+                            'currentMentees': [],
+                            'previousMentees': [],
+                        }
+                    }
+                }
+            )
+
+    def test_requested_fields_for_startup_mentor_relationship_type(self):
+        with self.login(email=self.basic_user().email):
+            obj = StartupMentorRelationshipFactory()
+            startup = obj.startup_mentor_tracking.startup
+            program = obj.startup_mentor_tracking.program
+            query = """
```

Comment:
This long literal string is a little awkward. Not a blocker, but I would prefer to have it at least externalized as a constant. Ideally, we'd write some code to generate this stuff, but that's not something we should hold this ticket up for.